### ISSUE TITLE
Split fermenter into two projects - fermenter-mda and fermenter-legacy-frameworks

### ISSUE THREAD

**Issue by d-ryan-ashcraft** [TechnologyBrewery/fermenter#2]
As Fermenter has grown, it's become common for it to be leveraged in open and closed frameworks.  Open frameworks are represented by the existing stout, ale, brett, and lambic frameworks that are currently co-located in the fermenter repository.  

At the same time, other users leverage Fermenter but don't open source their frameworks.  These users are interested in faster build times and more decoupling.  

One such decoupling example is removing the entity, service, etc concepts from being embedded in fermenter-mda to make it more clean to implement custom metamodel concepts.  

Additionally, frameworks like stout have arguably been superseded by opinionated frameworks like Spring Boot or Quarkus.  

Having fermenter-mda decoupled from frameworks allow the core, straightforward MDA engine to not be saddled with libraries that are now overcome by events, not desired in favor of other frameworks, etc.  

The vast majority of recent changes are solely related to fermenter-mda.

**Comment by d-ryan-ashcraft** [comment]
Found a minor documentation issue.  Cleaning that up and will close this ticket once the PR is approved.
